### DISCUSSION TITLE
Multiple choice question UI is not displayed correctly in apk

### DISCUSSION THREAD

**Discussion by swayangjit** [Sunbird-Knowlg/Community#184] [Bugs]
Using SDK 33, I have installed the release 4.6 apk.

Any course's user interface for assessment is improper.
We are unable to view the entire question because the questions are being trimmed.
![289453573-c8f96865-57a6-4df0-90c1-7afad21c486f](https://github.com/Sunbird-Knowlg/Community/assets/2860793/3625b66d-740f-4106-aa64-3c809a9e5640)
![289453703-bad6e8bc-5c78-49e7-a7a3-86010ab505e9](https://github.com/Sunbird-Knowlg/Community/assets/2860793/828d892c-7da5-4b8f-9fc0-8a14cd7b0fa3)

@HarishGangula  Can you please check this
[Reference](https://github.com/orgs/Sunbird-Ed/discussions/668)

**Comment by HarishGangula** [comment]
@swayangjit We can fix it by selecting the proper question template while creating the questions in the creation and test in different screen resolutions
